sdk/transaction/Cargo.toml

```diff
@@ -0,0 +1,75 @@
+[package]
+name = "solana-transaction"
+description = "Solana transaction-types"
+documentation = "https://docs.rs/solana-transaction"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+bincode = { workspace = true, optional = true }
+serde = { workspace = true, optional = true }
+serde_derive = { workspace = true, optional = true }
+solana-bincode = { workspace = true, optional = true }
+solana-feature-set = { workspace = true, optional = true }
+solana-frozen-abi = { workspace = true, optional = true }
+solana-frozen-abi-macro = { workspace = true, optional = true }
+solana-hash = { workspace = true }
+solana-instruction = { workspace = true }
+solana-logger = { workspace = true, optional = true }
+solana-precompiles = { workspace = true, optional = true }
+solana-program = { workspace = true }
```

Comment:
I think we'll be able to remove this and use `solana-message` instead, but then the features get a little tricky. Does it always enable the `bincode` feature for serialization, or does it enable the feature conditionally if `bincode` is enabled on this crate? That might require forcing `bincode` on this crate to be enabled if `verify` is enabled

Reply:
yeah `verify` requires `blake3` and `bincode`. And the `blake3` feature requires `bincode` anyway so we now have `verify = ["blake3", "solana-signature/verify"]`